Source/WebKit/WebProcess/WebPage/FindController.cpp

```diff
@@ -443,7 +443,10 @@ bool FindController::updateFindIndicator(LocalFrame& selectedFrame, bool isShowi
 
     m_findIndicatorRect = enclosingIntRect(indicator->selectionRectInRootViewCoordinates());
 #if PLATFORM(COCOA)
-    m_webPage->send(Messages::WebPageProxy::SetTextIndicator(indicator->data(), static_cast<uint64_t>(isShowingOverlay ? WebCore::TextIndicatorLifetime::Permanent : WebCore::TextIndicatorLifetime::Temporary)));
+    RefPtr rootFrame = &selectedFrame;
+    while (auto* parentFrame = dynamicDowncast<LocalFrame>(rootFrame->tree().parent()))
```

Comment:
Let's put this in Frame.cpp instead.  It needs to return a RefPtr because a RemoteFrame that is the main frame has no root frame.

Reply:
FrameTree.cpp?